src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  border: 1px solid $color-gray;
+  padding: 32px 16px 16px;
+
+  &__description {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-color;
```

Comment:
Ensure that the variable `$main-color` is defined in your SCSS files or imported correctly. Otherwise, it will cause a compilation error.